flink-table-store-connector/src/main/java/org/apache/flink/table/store/connector/TableStoreFactoryOptions.java

```diff
@@ -39,6 +39,14 @@ public class TableStoreFactoryOptions {
                                     + "By default, compaction does not adjust the bucket number "
                                     + "of a partition/table.");
 
+    public static final ConfigOption<String> COMPACTION_SCANNED_MANIFEST =
```

Comment:
Can you add format information in description? (how to serialize manifest entry to string?)